src/main/proto/th2_grpc_common/common.proto

```diff
@@ -83,6 +83,17 @@ message RawMessage {
   bytes body = 2;
 }
 
+message GroupMessage {
```

Comment:
The same here. I think that it will be a better name
```suggestion
message CombinedMessage {
```

Reply:
@OptimumOpium Do we still wanna do this renaming if we're leaving other names as they are? I'm not against it - just askin'

Reply:
I still believe that the `CombinedMessage` is a better name in this case because we have RawMessage and Message, and this one combines (aggregates) them. Maybe `AggragatedMessage` will be better than the previous suggestion.

Reply:
IMHO combination/aggregation implies that it holds both (raw and parsed) messages at the same time, which it doesn't. I'm gonna think of another, more suitable, name for this thing if we're against the current one

Reply:
It's just me who thinks that it is not suitable) If the name doesn't confuse anyone else we can leave it as it is.
@Nikita-Smirnov-Exactpro what do you think?

Reply:
Maybe use smth plain like `AnyMessage` or `RawOrParsedMessage`?

Reply:
It is not a so pretty name but it is more obvious for a user what he can get from it. `AnyMessage` will be good I think

Reply:
I think AggragatedMessage isn't the correct name because it contains full data related to the original message without any aggregation. CombinedMessage is more pretty but isn't obvious for a user.
We need a container to present one message transferred between client and server consist of several parts.

Reply:
Maybe cortege? It describes an ordered list of messages.

Reply:
@OptimumOpium @ivandruzhinin what do you think about MessageCortage?

Reply:
I dunno - `cortege` could be confusing to an end user too, because IMO it implies that I'll contain multiple messages, but in reality it'll contain only one